easylinks/src/main/java/com/google/step/servlets/DeleteLinkServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.google.step.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting links. */
+@WebServlet("/delete-link")
+public class DeleteLinkServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long id = Long.parseLong(request.getParameter("id"));
+    Key linkEntityKey = KeyFactory.createKey("Link", id);
+    DEFAULT_DATASTORE_SERVICE.delete(linkEntityKey);
```

Comment:
I guess here you need to do something like `ServletHelper.DEFAULT_DATASTORE_SERVICE`

Reply:
And also import the helper class

Reply:
Done.